src/HandBrake.php

```diff
@@ -0,0 +1,65 @@
+<?php
+declare(strict_types=1);
+namespace dje\AutoGourcer;
+
+/**
+ * Class HandBrake
+ * @package dje\AutoGourcer
+ */
+class HandBrake
+{
+
+    /**
+     * @var
+     */
+    public $path = './renders';
+
+    /**
+     * @var
+     */
+    public $file = '';
+
+    /**
+     * @var
+     */
+    public $ext = 'mp4';
+
```

Comment:
All of these `@var` docblocks are fairly useless :stuck_out_tongue:

Reply:
Noted; extra dock's never hurt...